build-system/server/amp4test.js

```diff
@@ -232,6 +232,29 @@ app.get('/a4a/:bid', (req, res) => {
   res.send(doc);
 });
 
+/**
+ * Serves an amp doc for test-shadow-amp.js
+ */
+app.get('/shadow-amp', (_req, res) =>
```

Comment:
This is an actual encoded test document, so ideally it'd be somewhere in fixtures. Do we have a place in fixtures where we can easily load any document on the test server?

Reply:
The other test documents seem to be loaded from this file.

Reply:
Hmm. Ok let's keep then since it's an established pattern. Though I think it's a bit painful.